index.html

```diff
@@ -25,7 +25,30 @@ <h2>Here are your results:</h2>
   <div id="add-js">
 
   </div>
+  
   <script src="js/script.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.1/lib/darkmode-js.min.js"></script>
```

Comment:
Do you think it might be less confusing to add these into a new file? Going for readability here.

Reply:
Thanks!